cloudflare.loader.php

```diff
@@ -10,14 +10,18 @@
 }
 
 // Rewrites Cloudflare IP
-$ipRewrite = new IpRewrite();
-
-$isCf = $ipRewrite->isCloudFlare();
-if ($isCf) {
-    // Fixes Flexible SSL
-    if (isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https') {
-        $_SERVER['HTTPS'] = 'on';
+try {
+    $ipRewrite = new IpRewrite();
+
+    $isCf = $ipRewrite->isCloudFlare();
+    if ($isCf) {
+        // Fixes Flexible SSL
+        if (isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https') {
+            $_SERVER['HTTPS'] = 'on';
+        }
     }
+} catch (\RuntimeException $e) {
+    error_log($e->getMessage());
```

Comment:
Thats okay.  If they look at their logs with any frequency they'll disable the plugin or fix their environment.